src/components/pages/searchpage/MainPage.js

```diff
@@ -0,0 +1,67 @@
+import React, { Component } from 'react';
+import PageName from "../../header/pagename/PageName";
+import FormTitle from "../../header/formtitle/FormTitle";
+import {HeaderCSSGrid, SearchCSSGrid} from "./MainPage.Styles";
+import SearchForm from "../../header/searchform/SearchForm";
+import {fetchDefault, fetchFromSearch} from "../../../util/dataLoader";
+import {sortingTypeForDisplay, sortingTypeForSearch} from "../../../util/sortingTypeConstants";
+import ResultsOptions from "../../helper/resultsoption/search/ResultOptions";
+import Results from "../../body/results/Results";
+import Constants from "../../constants/Constants";
+
+class MainPage extends Component {
+    state = {
+        data: '',
+        sortingType: Constants.RELEASE_DATE,
+        searchOption: Constants.TITLE
+    };
+
+    componentDidMount = () => {
+        fetchDefault(Constants.DEFAULT_LIMIT).then(data =>
+            this.setState(state => ({ data: data.data }))
+        );
+    };
+
+    changeSorting = data => {
+        this.setState(state => ({ sortingType: data }));
+    };
+
+    performSearch = searchString => {
+        const { sortingType, searchOption } = this.state;
+        fetchFromSearch(
+            searchString,
+            sortingTypeForSearch[sortingType],
+            searchOption
+        ).then(data => this.setState(state => ({ data: data.data })));
+    };
+
+    changeSearch = data => {
+        event.preventDefault();
```

Comment:
To be honest I haven't seen this global variable before). And I think it's not really good option for 2 reasons:
1. it's deprecated
2. I think it's better to keep all logic related to form implementation in form component. There you have access to regular event object and can prevent default to encapsulate logic and don't make consumers care about it

Reply:
done